### PR TITLE
Refactor CoreBundle Helpers to remove templating

### PR DESCRIPTION
**Pull request by mabumusa1**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. The functionality should not change after this PR. (we are removing Templating engine)
3. Try to test as much areas as you can specifically `Assets`, `Email`, `Menus`, `Builder`

### CODE REVIEW

File: app/bundles/LeadBundle/Entity/Import.php [hunk 1]
```diff
@@ -927,19 +925,4 @@ public function setIsPublished($isPublished)
 
         return parent::setIsPublished($isPublished);
     }
-
```
Comment (mabumusa1) [inline app/bundles/LeadBundle/Entity/Import.php:930]: @escopecz this function is not used anywhere, this is why I removed it

File: app/bundles/CoreBundle/Twig/Helper/ButtonHelper.php [hunk 2]
```diff
@@ -257,14 +261,15 @@ public function setLocation($location)
...
+     * @param string $buttonCount
      * @param string $groupType
      * @param null   $item
      *
      * @return $this
      */
     public function reset(Request $request, $buttonCount, $groupType = self::TYPE_GROUP, $item = null)
     {
+        //@escopecz: I think there is a possible bug here
         $this->location       = $buttonCount;
```
Comment (mabumusa1) [inline app/bundles/CoreBundle/Twig/Helper/ButtonHelper.php:273]: @escopecz the value that is stored in the location is `$buttonCount`, while we have ` setLocation` function, so this line is either necessary or could cause a bug that is not tested, because we are passing the wrong value

Comment (escopecz) [reply app/bundles/CoreBundle/Twig/Helper/ButtonHelper.php:273]: Do you have some steps to reproduce the bug? If not then TBH I don't want to get deep into this considering other priorities.

Comment (mabumusa1) [reply app/bundles/CoreBundle/Twig/Helper/ButtonHelper.php:273]: I do not have steps at the moment, let's close it for now and we will revisit if issues happen

File: app/bundles/AssetBundle/Controller/AssetController.php [hunk 3]
```diff
@@ -317,10 +315,6 @@ public function newAction($entity = null)
         $page   = $session->get('mautic.asset.page', 1);
         $action = $this->generateUrl('mautic_asset_action', ['objectAction' => 'new']);
 
-        // Get upload folder
-        $uploaderHelper = $this->container->get('oneup_uploader.templating.uploader_helper');
-        $uploadEndpoint = $uploaderHelper->endpoint('asset');
```
Comment (escopecz) [inline app/bundles/AssetBundle/Controller/AssetController.php]: @mabumusa1 We'll have to revert this. Assets are broken by this change. Why did you change it?

Comment (mabumusa1) [reply app/bundles/AssetBundle/Controller/AssetController.php]: I made the change based on the docs here https://github.com/1up-lab/OneupUploaderBundle/blob/master/doc/index.md

The goal is stop the usage if templating helper

Comment (escopecz) [reply app/bundles/AssetBundle/Controller/AssetController.php]: I don't get it. How is it related to the templating helper? The original code was creating a route. This new one is rendering a template.

Comment (mabumusa1) [reply app/bundles/AssetBundle/Controller/AssetController.php]: @escopecz you are right, the change is wrong. I will revert it now

Comment (mabumusa1) [reply app/bundles/AssetBundle/Controller/AssetController.php]: Resolved in commit 8053ea388de956e906845722136fac74a9d7fb40

File: app/bundles/AssetBundle/Resources/views/Asset/upload_endpoint.html.twig [hunk 4]
```diff
@@ -0,0 +1 @@
+{{ oneup_uploader_endpoint('asset') }}
```
Comment (escopecz) [inline app/bundles/AssetBundle/Resources/views/Asset/upload_endpoint.html.twig]: I guess this can be removed now? @mabumusa1

Comment (escopecz) [reply app/bundles/AssetBundle/Resources/views/Asset/upload_endpoint.html.twig]: I'll remove it.

**Review by escopecz — APPROVED** [review]
Ok, I went through the code changes and did not notice anything clocking. There is some missing space here, unnecessary comment there but nothing that's worth a comment.

I clicked through the UI. Especially email and asset and did not notice any more issues.

**Comment by biozshock** [comment]
Noticed another place where theme helper is used: 
https://github.com/mautic/mautic/blob/5.x/app/bundles/CoreBundle/Controller/ExceptionController.php#L74

**Comment by escopecz** [comment]
I'll merge this even without a second approval as this is requirement to get us to the Symfony upgrade. Too bad it wasn't double-tested.
